src/sentry/models/organization.py

```diff
@@ -439,10 +439,12 @@ def handle_2fa_required(self, request):
             ip_address=ip_address
         )
 
-    def get_url(self):
+    def get_url_viewname(self):
         from sentry import features
         if features.has('organizations:sentry10', self):
-            url = reverse('sentry-organization-issue-list', args=[self.slug])
+            return 'sentry-organization-issue-list'
         else:
-            url = reverse('sentry-organization-home', args=[self.slug])
-        return url
+            return 'sentry-organization-home'
+
+    def get_url(self):
+        return reverse(self.get_url_viewname(), args=[self.slug])
```

Comment:
Will it be ok if emails continue using get_url()?

Reply:
Are emails in getsentry or sentry?

Reply:
Nm literally can't read. I think that `get_url` will not work at all in `getsentry`, unless there's some magic there that I missed.